tests/regressionTests/inputs/slabBurner2D/slabBurner2DMeshGen.py

```diff
@@ -75,38 +76,60 @@ def defineBoundary(sides, name, boundary_list):
 gmsh.model.geo.synchronize()
 
 # set the default properties to generate quad mesh
-gmsh.option.setNumber("Mesh.Algorithm", 8)  # 8: Frontal-Delaunay for Quads
+gmsh.option.setNumber("Mesh.Algorithm", 11)  # 11: Quasi-structured Quad
 gmsh.option.setNumber("Mesh.Algorithm3D", 1)  # 1: Delaunay
 gmsh.option.setNumber("Mesh.RecombinationAlgorithm", 3)  # 3: blossom full-quad
-gmsh.option.setNumber("Mesh.MeshSizeMin", 0.0005)
-gmsh.option.setNumber("Mesh.MeshSizeMax", 0.001)  # assume about 1mm resolution
+gmsh.option.setNumber("Mesh.MeshSizeMin", 0.003)
+gmsh.option.setNumber("Mesh.MeshSizeMax", 0.0035)  # with the other options this results in about 0.6 mm element size
 gmsh.option.setNumber("Mesh.SubdivisionAlgorithm", 1)  # 1: all quadrangles
 gmsh.option.setNumber("Mesh.RecombineAll", 1)  # true
 
 # set the options to prevent gmsh from adding too many elements to each geometry line
 gmsh.option.setNumber("Mesh.MeshSizeFromPoints", 0)
 gmsh.option.setNumber("Mesh.MeshSizeFromCurvature", 0)
 gmsh.option.setNumber("Mesh.MeshSizeExtendFromBoundary", 0)
+gmsh.option.setNumber("Mesh.Smoothing", 10)
 
 # generate the mesh
 gmsh.model.mesh.setRecombine(2, surface_id)
 gmsh.model.mesh.generate(2)
-gmsh.model.mesh.reverse() # we have to reverse the element orientation for petsc
 
```

Comment:
Do you have to sync again here?

Reply:
I think that gmsh.model.geo.synchronize() is only used for the cad.  https://gitlab.onelab.info/gmsh/gmsh/blob/master/tutorials/python/t1.py States that ```Before they can be meshed (and, more generally, before they can be used by API functions outside of the built-in CAD kernel functions)```

Reply:
Okay great I thought you have to use them every time the gmsh object is called.  Thank you.